pineappl_py/src/import_only_subgrid.rs

```diff
@@ -2,10 +2,75 @@ use super::subgrid::PySubgridEnum;
 
 use numpy::{PyReadonlyArray1, PyReadonlyArray3};
 use pineappl::import_only_subgrid::ImportOnlySubgridV1;
+use pineappl::import_only_subgrid::ImportOnlySubgridV2;
 use pineappl::sparse_array3::SparseArray3;
-
+use pineappl::subgrid::Mu2;
 use pyo3::prelude::*;
 
+/// PyO3 wrapper to :rustdoc:`pineappl::import_only_subgrid::ImportOnlySubgridV2 <import_only_subgrid/struct.ImportOnlySubgridV2.html>`
+///
+/// **Usage**: `pineko`
+#[pyclass]
+#[derive(Clone)]
+#[repr(transparent)]
+pub struct PyImportOnlySubgridV2 {
+    pub(crate) import_only_subgrid: ImportOnlySubgridV2,
+}
+
+impl PyImportOnlySubgridV2 {
+    pub(crate) fn new(import_only_subgrid: ImportOnlySubgridV2) -> Self {
+        Self {
+            import_only_subgrid,
+        }
+    }
+}
+
+#[pymethods]
+impl PyImportOnlySubgridV2 {
+    #[new]
+    pub fn new_import_only_subgrid(
```

Comment:
Call this constructor simply `new`.

Reply:
If I do this it complains that there are two methods called new. I believed I have done some mess with the declarations above

Reply:
I should've been clearer:

1. https://github.com/NNPDF/pineappl/pull/222/files#r1179250872
2. then rename the constructor to `new`. At the very least you should get another error message (but hopefully none).

Reply:
Try replacing this line with

```rust
Self {
    import_only_subgrid: ImportOnlySubgridV2::new(
        // ..
    )
}
```

The `Self { ... }` syntax is how you construct an object if you have access to its members (if you're in the same module or if all members are visible to you).

Reply:
Thanks! Addressed in [605fd90](https://github.com/NNPDF/pineappl/pull/222/commits/605fd90d088b4c6a384dac6b0c162460328393e7)